app/services/cascade_assets_client.rb

```diff
@@ -0,0 +1,86 @@
+require 'savon'
+
+module CascadeAssetsClient
+  CASCADE_ASSET_OPERATION_WSDL = (Rails.application.secrets.cascade_asset_operation_endpoint + "?wsdl").freeze
+  SITE_NAME = "Chapman.edu".freeze
+  ASSETS_PATH = "_assets/".freeze
+  ASSET_BLOCK_PATH = "_cascade/blocks/html/cascade-assets/".freeze
+
+  @client = Savon.client(wsdl: CASCADE_ASSET_OPERATION_WSDL)
+
+  module_function
+
+  def request(operation, message)
+    @client.call(operation, message: message)
+    rescue Savon::Error => error
+      msg = "Cascade web service responded with %s: %s"
+      Rails.logger.error format(msg, error.http.code, error.message)
+      nil
+  end
+
+  def read_asset_block(block_path)
+    asset_identifier = {
+      path: {
+        path: block_path,
+        siteName: SITE_NAME
+      },
+      type: "block"
+    }
+
+    message = {authentication: deploy_credentials, identifier: asset_identifier}
+    request(:read, message)
+  end
+
+  def write_asset_block(local_block)
+    # First get a reference to the remote asset block, and then update the xml with the local block
+    read_response = read_asset_block(ASSET_BLOCK_PATH)
+    return unless read_response && read_response.body[:read_response][:read_return][:success]
```

Comment:
I worry about unexpected silent failures here. Seems like we might want to log this or raise an exception.